src/mediaplayer.cs

```diff
@@ -2085,5 +2121,49 @@ interface MPMusicPlayerApplicationController
 		[Export ("performQueueTransaction:completionHandler:")]
 		void Perform (Action<MPMusicPlayerControllerMutableQueue> queueTransaction, Action<MPMusicPlayerControllerQueue, NSError> completionHandler);
 	}
+
+	[NoTV][NoMac]
+	[iOS (11,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MPMusicPlayerPlayParameters : NSSecureCoding {
+		[Export ("initWithDictionary:")]
+		IntPtr Constructor (NSDictionary<NSString, NSObject> dictionary);
```

Comment:
If someday we can strongify this API at some point doing `NSDictionary<NSString, NSObject>` instead of just `NSDictionary` will ease the transition a lot, we will avoid casting issues